catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -486,6 +486,16 @@ private void addMetacardDataQuality(MetacardImpl metacard) {
         try (InputStream inputStream = getSourceInputStream()) {
             DocumentBuilderFactory domFactory = DocumentBuilderFactory.newInstance();
             domFactory.setNamespaceAware(false);
+            try {
+                domFactory.setFeature(
+                        "http://apache.org/xml/features/nonvalidating/load-dtd-grammar",
+                        false);
+                domFactory.setFeature(
+                        "http://apache.org/xml/features/nonvalidating/load-external-dtd",
+                        false);
```

Comment:
I may be wrong, but I don't think we would ever be dealing with DTD xml, so I think this is OK to disable.